test/WebJobs.Script.Tests.Integration/TestFunctionHost.cs

```diff
@@ -172,6 +187,39 @@ public async Task RestartAsync(CancellationToken cancellationToken)
             await _hostService.RestartHostAsync(cancellationToken);
         }
 
+        private void StillRunningCallback(object state)
```

Comment:
This is a general-purpose log that will get written to the console (so, visible in DevOps) if a host is continuously running in the background. It should give us enough details to figure out what host it is.